libcloud/loadbalancer/drivers/elb.py

```diff
@@ -308,8 +308,7 @@ def ex_create_balancer_listeners(self, name, listeners=None):
             if protocol == 'HTTPS' or protocol == 'SSL':
                 params['Listeners.member.%d.   \
                         SSLCertificateId' % i] = listener[3]
-        else:
-            return False
+        return False
```

Comment:
do you want me to revert that change or are you happy to leave it in since it doesn't change the current behaviour?